edm4hep.yaml

```diff
@@ -339,9 +355,9 @@ datatypes :
       - edm4hep::ObjectID  rawHits     //raw data hits. Check getType to get actual data type.
 
 
- #----------  TPCHit
-  edm4hep::TPCHit:
-    Description: "Time Projection Chamber Hit"
+ #---------- TrackerRawData
+  edm4hep::TrackerRawData:
```

Comment:
```suggestion
  edm4hep::ADCTimeSeries:
```

Reply:
```diff
- - int32_t rawDataWords //raw data (32-bit) word at i.
+ - int32_t adcCounts    //raw data (32-bit) word at i.

Reply:
Need an entry for interval?

Reply:
I am wondering whether `ADCTimeSeries` is not making this too specific? (In this case it is one, but in other cases a more general concept might be better, instead of having to introduce another type later?)

Reply:
> I am wondering whether `ADCTimeSeries` is not making this too specific? (In this case it is one, but in other cases a more general concept might be better, instead of having to introduce another type later?)

How about using "RawTimeSeries"?

Reply:
> Need an entry for interval?

I think it is needed.